src/patches.rs

```diff
@@ -12807,6 +12855,51 @@ fn build_and_run_patches(gc_disc: &mut structs::GcDisc, config: &PatchConfig, ve
                 move |res| patch_artifact_totem_scan_strg(res, &strg_text),
             );
         }
+
+        const LORE_SCAN_STRGS: &[ResourceInfo] = &[
+            // Choze Lore
+            resource_info!("0q_connectChozo Lore 001.STRG"),    // Fountain
+            resource_info!("Chozo Lore 002.STRG"),              // Exodus
+            resource_info!("Chozo Lore 003.STRG"),              // Meteor Strike
+            resource_info!("Chozo Lore 004.STRG"),              // Contain
+            //resource_info!("Chozo Lore 006.STRG"),              // Newborn
+            resource_info!("Chozo Lore 007.STRG"),              // Cipher
+            resource_info!("Chozo Lore 008.STRG"),              // The Turned
+            resource_info!("Chozo Lore 009 scan.STRG"),         // Infestation
+            resource_info!("Chozo Lore 011 scan.STRG"),         // Worm
+            resource_info!("Chozo Lore 012.STRG"),              // Hope
+            resource_info!("Chozo Lore 012 scan.STRG"),         // Hatchling's Shell
+            resource_info!("Chozo Lore 013.STRG"),              // Binding
+            resource_info!("Chozo Lore 014.STRG"),              // Beginnings
+            resource_info!("Chozo Lore 015.STRG"),              // Statuary
+            //resource_info!("16_Furnances Chozo Lore.STRG"),     // Cradle
+            resource_info!("Chozo Lore 017.STRG"),              // Hatchling
+
+            // Pirate Logs
+            resource_info!("Pirate log 004.STRG"),   // Phazon Analysis
+            resource_info!("Pirate log 005.STRG"),   // Phazon Program
+            resource_info!("Pirate log 008.STRG"),   // Mining Status
+            resource_info!("Pirate log 010.STRG"),   // Security Breaches
+            resource_info!("Pirate log 011.STRG"),   // Meta Ridley
+            resource_info!("Pirate log 013.STRG"),   // The Hunter
+            resource_info!("Pirate log 014.STRG"),   // Phazon Infusion
+            resource_info!("Pirate log 015.STRG"),   // Metroid Prime
+            resource_info!("Pirate log 016.STRG"),   // Prime Breach
+            resource_info!("Pirate log 017.STRG"),   // Prime Mutations
+            // resource_info!("Pirate log 021.STRG"),   // Metroid Morphology
+            // resource_info!("Pirate log 022.STRG"),   // Metroid Forces
+            //resource_info!("Pirate log 024.STRG"),   // Hunter Weapons
+            //esource_info!("Pirate log 025.STRG"),   // Chozo Studies
+            //resource_info!("Pirate log 026.STRG"),   // Chozo Artifacts
+            //resource_info!("Pirate log 027.STRG"),   // Chozo Ghosts
+        ];
+        for (res_info, strg_text) in LORE_SCAN_STRGS.iter().zip(lore_strings.iter()) {
```

Comment:
There may be cases where we only want to patch some lore scans but not others. This loop here makes that difficult.